### ISSUE TITLE
Autocomplete: `aria-labelledby` is still present when `aria-label` is passed.

### ISSUE THREAD

**Issue by catamphetamine** [downshift-js/downshift#1182]
Autocomplete: `aria-labelledby` is still present when `aria-label` is passed.
Both on `getMenuProps()` and `getInputProps()`.
Workaround:

```js
        <input
          ref={ref}
          {...getInputProps({
            'aria-labelledby': undefined,
            'aria-label': 'abc'
          })} />
```

**Comment by silviuaavram** [comment]
That's expected. I understand that it may be cool to have `aria-labelledby` removed automatically when `aria-label` is used. However it's not something I would like to encourage. Form elements should always have a physical label for many reasons. If your spec simply just cannot contain a physical label, then the workaround above should be good enough.

Thank you!
